### DISCUSSION TITLE
引用\cite问题

### DISCUSSION THREAD

**Discussion by jhl13** [mengchaoheng/SCUT_thesis#16] [General]
你好，请问怎么设置引用平齐？模板里使用\cite直接变成了右上角。

**Comment by mengchaoheng** [comment]
> 你好，请问怎么设置引用平齐？模板里使用\cite直接变成了右上角。

请仔细看模版说明，有讲

**Reply by jhl13** [reply]
找到了，谢谢，还想问一个问题，我看模板生成出来的目录里面没有目录这一项，请问这个是正常的吗？

**Reply by mengchaoheng** [reply]
你这个问题问得好啊，这个可把我们几个开发者搞的昏天地暗了。我之前努力了很久把目录做好，里面有目录。后面有同学说不应该有，然后找了很久才找到方法去掉了。

> https://github.com/mengchaoheng/SCUT_thesis/issues/3#issue-1147769602

**Reply by mengchaoheng** [reply]
还有同学说每次都手动删掉目录里的目录。这个到底应不应该有，看来得开个讨论会投票一下

**Reply by mengchaoheng** [reply]
@jhl13
